net/heartbeat.py

```diff
@@ -32,9 +32,9 @@ def __init__(self, only_ip="127.0.0.1"):
         LoopingCall(self._heartbeat).start(10, now=True)
 
     def buildProtocol(self, addr):
-        if self.status in ("starting up", "generating GUID") and self.only_ip != "127.0.0.1":
+        if self.status in ("starting up", "generating GUID") and "127.0.0.1" not in self.only_ip:
```

Comment:
Yes, I see how that check does not do what you had intended.  I have updated to do so I believe.